native/src/main/java/tech/sourced/babelfish/TranslationUnitJSONMapper.java

```diff
@@ -10,20 +10,21 @@
 import com.fasterxml.jackson.databind.module.SimpleModule;
 import org.eclipse.cdt.core.dom.ast.IASTTranslationUnit;
 
-import java.io.ByteArrayOutputStream;
+import java.io.OutputStream;
+import java.io.PrintStream;
 import java.io.IOException;
 
 class TranslationUnitJSONMapper implements IExchangeFormatWritter {
 
     final JsonGenerator generator;
     final JsonFactory jsonFactory = new JsonFactory();
     final ObjectMapper mapper = new ObjectMapper();
-    private ByteArrayOutputStream byteOutputStream;
+    private PrintStream printStream;
```

Comment:
```suggestion
    private OutputStream printStream;
```
To match an output of `getOutputStream` method.

Reply:
Yes, I started using that type but switched to the abstract one since it was more idiomatic, thanks for the catch!